packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -1,31 +1,56 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import puppeteer, { Browser, Page } from 'puppeteer';
-import { dataUiId } from '../utils';
+import path from 'path';
+import dotenv from 'dotenv';
+import { chromium, Browser, Page } from 'playwright';
+import faker from 'faker';
+import { dataUiId, createUserAndThread, encodeQueryData, getNameInitials } from '../utils';
 import { startServer, stopServer } from './app/server';
 import { CHAT_UI_IDS } from '../../../src';
-import { COMPONENT_UI_IDS } from '../../../../react-components';
+import { COMPONENT_UI_IDS } from '@internal/react-components';
+
+dotenv.config({ path: path.join(__dirname, '..', '.env') });
+
+jest.setTimeout(30000);
+
+const CONNECTION_STRING = process.env.CONNECTION_STRING ?? '';
+const TOPIC_NAME = 'Cowabunga';
 
 const SERVER_URL = 'http://localhost:3000';
+
 const PAGE_VIEWPORT = {
   width: 1200,
-  height: 768,
-  deviceScaleFactor: 1
+  height: 768
 };
 
+const MAX_PAGES = 2;
+
 let browser: Browser;
-let page: Page;
+const pages: Array<Page> = [];
+const participants: Array<string> = [];
 
-describe('Chat Composite Puppeteer Tests', () => {
+describe('Chat Composite E2E Tests', () => {
   beforeAll(async () => {
     await startServer();
```

Comment:
This file will need to be split up across different files if we are to add lots more tests, will we need to call startServer in each file? If so can this be moved to a global setup (perhaps: https://jestjs.io/docs/configuration#globalsetup-string?)

Reply:
I dislike magic.

I like the fact that, right here in the test, it says what happens before the tests run. So my recommendation would be: do nothing right now. When we split the tests into multiple files, extract a function into a helper module, that still gets called in `beforeAll` block in each test suite.

Reply:
Thanks for the call out. Let's refactor when we reach that point.

Reply:
It would be great if splitting into separate files could be a fast follow to this PR! Folks writing tests shouldn't have to figure the part of how to split it up as new files and as it stands that file is gonna get real huge real fast (with only 4 tests its already over 100 lines long)